release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -100,17 +100,45 @@ For more information, see xref:../updating/preparing_for_updates/updating-cluste
 [id="ocp-4-16-web-console_{context}"]
 === Web console
 
+[id="ocp-4-16-web-console-language"]
+==== Language support for French and Spansih
+With this release, French and Spanish are suported in the web console. You can update the language in the webconsole from the *Language* list on the *User Preferences* page.
+
+[id="ocp-4-16-patternfly4-deprecated"]
+==== Patternfly 4 is now deprecated with 4.16
+With this release, Patternfly 4 and React Router 5 is deprecated in the web console. All plugins should migrate to Patternfly 5 and React Router 6 as soon as possible.
+
 [id="ocp-4-16-administrator-perspective_{context}"]
 ==== Administrator perspective
 
 This release introduces the following updates to the *Administrator* perspective of the web console:
-//Are there any for 4.16
+
+* A Google Cloud Platform (GCP) token authorization `Auth Token GCP`, and a `Configurable TLS ciphers` filter was added to the *Infrastructure features* filter in the OperatorHub.
+* A new quick start, *Build with guided documentation*, is available with information on impersonating the `system:admin` user.
```

Comment:
<img width="375" alt="Screenshot 2024-06-18 at 11 19 25 AM" src="https://github.com/openshift/openshift-docs/assets/12692381/70d53039-937b-4b57-a26c-6676e3ad3187">
The new quick start name is `Impersonating the system:admin user`